packages/terra-alert/package.json

```
@@ -1,7 +1,7 @@
 {
   "name": "terra-alert",
   "main": "lib/Alert.js",
-  "version": "4.60.6",
+  "version": "4.60.7",
```

Shouldn't this be:
```suggestion
  "version": "4.61.0",
```
since terra-icon had a minor dependency bump